### DISCUSSION TITLE
Can you add a Bhopping community since this is built off of Quake engine?

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [nzp-team/nzportable#1004] [General]
I think this is **AWESOME!** that this is built off of Quake, theoretically, if you added stuff like Bhopping, and airstrafing mechanics back. (But in like a different game mode) I think it would gain more attraction, Bhoping is very popular, so maybe just think about it. I would love this game a lot more if you added that.

**Comment by MotoLegacy** [comment]
I think one of the coolest things about NZ:P is that despite the Quake backbone playing it feels next to nothing like Quake. We should be looking forward and not back, the goals of this project are clearly outlined. :)

**Reply by Charlieis1nsan3** [reply]
Yeah but, I was thinking like another game mode so you can still obviously play normally, but their is another game mode.

**Reply by Charlieis1nsan3** [reply]
okay 😞

**Comment by Peter0x44** [comment]
I think that would be cool, I would love a standalone quake-based surfing or bhop game. But NZP is not the project to implement it in, and it doesn't align with our goals.

**Reply by Charlieis1nsan3** [reply]
their is one. but their is kinda only 2 that's why it would be nice to have another.
https://quake2.m-h.org.uk/
https://quake.m-h.org.uk/
